src/arb/Complex.jl

```diff
@@ -1616,135 +1616,127 @@ end
 #
 ################################################################################
 
-for (typeofx, passtoc) in ((ComplexFieldElem, Ref{ComplexFieldElem}), (Ptr{ComplexFieldElem}, Ptr{ComplexFieldElem}))
```

Comment:
Getting rid of this loop by replacing `$typeofx ` with `ComplexFieldElemOrPtr` and `$passtox` with `Ref{ComplexFieldElem}` is also something we could re-apply (and similar in the other files, of course)

Reply:
agreed. I'll try to do this with as less conflicts as possible